yoti-sdk-api/src/main/java/com/yoti/api/client/qrcode/extension/LocationConstraintContent.java

```diff
@@ -0,0 +1,59 @@
+package com.yoti.api.client.qrcode.extension;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LocationConstraintContent {
+
+    private final DeviceLocation expectedDeviceLocation;
+
+    LocationConstraintContent(double latitude, double longitude, double radius, double maxUncertainty) {
+        this.expectedDeviceLocation = new DeviceLocation(latitude, longitude, radius, maxUncertainty);
+    }
+
+    @JsonProperty("expected_device_location")
+    public DeviceLocation getExpectedDeviceLocation() {
+        return expectedDeviceLocation;
+    }
+
+    public static final class DeviceLocation {
+
+        private final double latitude;
+        private final double longitude;
+        private final double radius;
+        private final double maxUncertainty;
```

Comment:
Mick please consider radius and maxRadius are optional with default value 150 mt. You'd probably like to allow the client to specify only lat and lon

Reply:
Good spot.  I shall fix that...